laws/src/main/scala/cats/laws/discipline/Eq.scala

```diff
@@ -31,17 +32,9 @@ object eq {
       (a: A) => showInstance.show(a)
     }
 
-  // Temporary, see https://github.com/non/algebra/pull/82
-  implicit def tuple2Eq[A, B](implicit A: Eq[A], B: Eq[B]): Eq[(A, B)] =
-    new Eq[(A, B)] {
-      def eqv(x: (A, B), y: (A, B)): Boolean =
-        A.eqv(x._1, y._1) && B.eqv(x._2, y._2)
-    }
+  implicit def tuple2Eq[A, B](implicit A: Eq[A], B: Eq[B]): Eq[(A, B)] = tuple.tuple2Eq
```

Comment:
We should be able to just remove these right? And do the correct import at the places that use this?